libs/utils/sanitizeHtml.js

```diff
@@ -0,0 +1,37 @@
+function stringToHTML(str) {
+  const parser = new DOMParser();
```

Comment:
If `str` is not a TrustedHTML, the next line will throw an error. Let's add the try catch statement or checking if `str` is defined and is a proper data type